WCSimFLOWER.cpp

```diff
@@ -53,6 +53,42 @@ WCSimFLOWER::WCSimFLOWER(const char * detectorname, WCSimRootGeom * geom, bool o
     fTopBottomDistanceLo = 2670;
     fTopBottomDistanceHi = 2690;
     break;
+  case kHyperK20BnL0mPMT: // WCSim Hybrid 20% case
+    fDarkRate = 8.4;  // dark rate of 50cm Box&Line PMTs
+    fDarkRate2 = 0.; // dark rate of 7.5cm PMTs in mPMT modules
+    fNPMTs = 19418;   // total number of 50cm Box&Line PMTs
+    fNPMTs2 = 0; // total number of 7.5cm PMTs (10k mPMT modules, 19 PMTs each)
+    fNeighbourDistance = 145;
+    fTopBottomDistanceLo = 2670;
+    fTopBottomDistanceHi = 2690;
+    break;
+  case kHyperK40BnL0mPMT:
+    fDarkRate = 8.4;  // dark rate of 50cm Box&Line PMTs
+    fDarkRate2 = 0.; // dark rate of 7.5cm PMTs in mPMT modules
+    fNPMTs = 38952;   // total number of 50cm Box&Line PMTs
+    fNPMTs2 = 0; // total number of 7.5cm PMTs (10k mPMT modules, 19 PMTs each)
+    fNeighbourDistance = 145;
```

Comment:
I think this should be closer* to 102, like the kHyperK40 case

*Since the hybrid geometry is slightly different to the normal geometry, it may be that 102 & 145 aren't quite right

Reply:
Ah yes, I didn't check what should be the correct value for fNeighbourDistance.